common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -69,7 +37,7 @@ export function sourcemapsExtra() {
 export type WarningInhibitor = (warning: RollupWarning) => boolean;
 
 function matchesPathSegments(str: string | undefined, segments: string[]): boolean {
-  return !str ? false : str.includes(segments.join("/")) || str.includes(segments.join("\\"));
+  return str?.includes(segments.join(path.sep)) ?? false;
```

Comment:
This might so sounds strange but path.sep was broken on Windows in some configuration because warnigs could come back are using both "/" and "\".  Thus PR #24386.

Reply:
Thanks for clarifying! I addressed it in https://github.com/Azure/azure-sdk-for-js/pull/27285/commits/2992192a8458a451d14b90a64cd55b6c5bec8df2.